fix(glob): fixed calls to glob

daragh-king-genesys opened the pull request:
fixed calls to glob

COMUI-1703

katie-bobbe-genesys commented:
Do we know why the calls to `glob` stopped working as expected?

gavin-everett-genesys commented on packages/genesys-spark-components/src/components/stable/gux-icon/icons/legacy/reset-view.svg:
Was this to be included also?

daragh-king-genesys replied:
Ya the fix was made when I was testing the script so might as well keep it.

daragh-king-genesys commented:
> Do we know why the calls to `glob` stopped working as expected?

When upgrading from globV8 -> v10 I missed two places where we were using callbacks that were no longer supported in v10.
I have fixed them and also updated all synchronous calls to glob to use the same method `globSync`.

katie-bobbe-genesys commented:
I cannot verify the changes on the feature branch because the `spark-monorepo-examples` Jenkins job failed on the `Generate Asset Manifest` stage 
![image](https://github.com/MyPureCloud/genesys-webcomponents/assets/82100934/16c4da61-8f1c-49bc-9d44-2f9e3abd989e)
